virttest/qemu_vm.py

```diff
@@ -36,6 +36,21 @@
 from virttest.qemu_devices import qdevices, qcontainer
 
 
+def traverse_nested_dict(d):
```

Comment:
Suggest to move this function into `qemu_monitor.py` as an utility, since users do have other needs to handle this kind of QMP data.